datascroller/scroller.py

```diff
@@ -234,9 +234,20 @@ def _display_string_using_curses(self, screen, otherstring):
     # TODO(baogorek): figure out what to do with function above
     def _add_string_using_curses(self, screen, otherstring):
         """Prints strings for use with the scroller"""
+
+        # only necessary if we want to use color. First argument is an ID for
+        #   this pairing. Second and third are the text and background colors
+        #   respectively
+        curses.init_pair(1, curses.COLOR_CYAN, curses.COLOR_BLACK);
```

Comment:
We could put the two colors in the "poor man's config" section and remove the comments above this line. Eventually we'll want to give color customization but I think these colors are a clear improvement from the default.

Reply:
Great! I'll move the initialization. I think I'll keep the instructional comment about the arguments, though, just to help us / others remember while we're figuring all this out.